examples/sdk-snapgrid/src/components/MasonryGrid/MasonryGrid.module.css

```
@@ -0,0 +1,318 @@
+  /*
+   * FIX 1: overflow:hidden is REMOVED from .card.
+   * It was causing CSS columns to break inside cards.
+   * Clipping now lives only on .imageWrap where it belongs.
+   */
+
+  /* entry animation */
+  opacity: 0;
+  animation: cardIn 0.5s cubic-bezier(0.22, 1, 0.36, 1) forwards;
+}
+
+@keyframes cardIn {
```

Stylelint (17.9.0)

[error] Expected keyframe name "cardIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)